packages/react-experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestions.tsx

```diff
@@ -68,7 +69,7 @@ export const BaseFloatingSuggestions = <T extends {}>(props: IBaseFloatingSugges
         <Callout
           className={classNames.callout}
           isBeakVisible={false}
-          gapSpace={5}
+          gapSpace={gapSpace || 5}
```

Comment:
>gapSpace || [](http://example.com/codeflow?start=20&length=11)

I think `0` should be a valid gap space if the caller explicitly specifies no gap, so maybe use `gapSpace ?? 5` here?

Reply:
thanks for catching this, I always forget about 0 in these cases.